CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 2.17.0-rc.0 / 2020-03-11
+
+This release implements isolation in TSDB. API queries and recording rules are guaranteed to only see full scrapes and full recording rules. This causes a slight increase in memory usage.
```

Comment:
"…see full scrapes and full recording rules." → "…see the full result of a scrape or rule evaluation."

"This causes a slight increase in memory usage." → It's not only memory usage. Perhaps something like: "This comes with a certain overhead in resource usage. Depending on the situation, there might be a noticeable increase in memory usage, CPU usage, or query latency."

Reply:
Indeed. Updated.